### ISSUE TITLE
Parser: Implement basic `CREATE TABLE` DDL

### ISSUE THREAD

**Issue by KieranKaelin** [cybertec-postgresql/poc-plpgsql-analyzer#58]
In order to be able to succeed the ANTLR4 parser used in the Atroposs DMA Module, the parser must be able to understand DDL. This is the first of many issues marking the way to replacing said parser.

## Goal

A basic `CREATE TABLE` statement shall produce an appropriate AST.

For reference, see the [Oracle documentation](https://docs.oracle.com/en/database/oracle/oracle-database/21/sqlrf/CREATE-TABLE.html#GUID-F9CE0CC3-13AE-4744-A43C-EAC7A71AAAB6).

### Examples

```sql
-- SQL Developer Export
CREATE TABLE "MY_SCHEMA"."EMP" (
  "EMP_ID" NUMBER(6,0) DEFAULT MY_SCEHMA.EMP_SEQ.nextval,
  "CREATED_AT" TIMESTAMP WITH TIME ZONE,
  "EMAIL" VARCHAR2(25 BYTE)
 )
SEGMENT CREATION IMMEDIATE
ORGANIZATION INDEX NOCOMPRESS PCTFREE 10 INITRANS 2 MAXTRANS 255 LOGGING
STORAGE(INITIAL 65536 NEXT 1048576 MINEXTENTS 1 MAXEXTENTS 2147483645
PCTINCREASE 0 MAXSIZE UNLIMITED FREELISTS 1 FREELIST GROUPS 1
BUFFER_POOL DEFAULT FLASH_CACHE DEFAULT CELL_FLASH_CACHE DEFAULT)
TABLESPACE "SYSAUX"
PCTTHRESHOLD 50;
```

## How to demo

Appropriate tests are created showcasing the resulting AST **and** AST accessors to iterate and inspect the table properties.

## Non-goals

- Constraints: The SQL Developer exports them in separate DDL statements, never as inline constraints
- Identity columns: Share syntax with sequences, and may be implemented in parallel
- Partitions: Implemented as part of another ticket due to their complexity
